app/controllers/state_file/questions/primary_state_id_controller.rb

```diff
@@ -0,0 +1,26 @@
+module StateFile
+  module Questions
+    class PrimaryStateIdController < QuestionsController
+      include ReturnToReviewConcern
+
+      def edit
+        super
+        render "state_file/questions/#{current_state_code}_primary_state_id/_#{current_state_code}_primary"
+      end
+
+      def update
+        @form = initialized_update_form
+        if @form.valid?
+          @form.save
+          after_update_success
+          track_question_answer
+          redirect_to(next_path)
+        else
+          after_update_failure
+          track_validation_error
+          render "state_file/questions/#{current_state_code}_primary_state_id/_#{current_state_code}_primary"
+        end
```

Comment:
[dust] oof, is thia side effect of pulling the render into the controller? sorry about that. :/

Reply:
i dont hate it ?

Reply:
we could extract lines 8 & 21 into a a method named something like `state_specific_view`